app/views/participant_signup/_contact.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= f.inputs :name => "Contact Information" do -%>
```

Comment:
You could just do `<%= field :bad_address unless @current_user.person_type == 'Participant' %>` maybe?